lib/add_epoch_column.py

```diff
@@ -0,0 +1,21 @@
+from pyspark.sql.functions import col, when
+from pyspark.sql import DataFrame
+
+def add_epoch_column(df: DataFrame, periods = [1901,1918,1926,1939,1954,1970,1985,1994,2009]) -> DataFrame:
```

Comment:
propozycja. 
Utworzyć zmienną 
```
DEFAULT_PERIODS = [1901,1918,1926,1939,1954,1970,1985,1994,2009]
```
A samą metodę zrobić tak, że 
```
def add_epoch_column(df: DataFrame, periods : Optional[List[int]] = None) -> DataFrame:
if periods is None:
  periods = DEFAULT_PERIODS
```

Reply:
zrobione

Reply:
Nie to nie jest to o co poprosiłem. Zmienna powinna być w zakresie globalnym pliku i mieć nazwę zarezerwowaną dla stałych w kodzie.

Reply:
poprawiłem